pkg/graph/resolvers/trb_request_feedback.go

```diff
@@ -0,0 +1,87 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/storage"
+)
+
+// CreateTRBRequestFeedback creates a TRBRequestFeedback in the database
+func CreateTRBRequestFeedback(ctx context.Context, store *storage.Store, feedback *models.TRBRequestFeedback) (*models.TRBRequestFeedback, error) {
+	feedback.CreatedBy = appcontext.Principal(ctx).ID()
+
+	form, err := store.GetTRBRequestFormByTRBRequestID(ctx, feedback.TRBRequestID)
+	if err != nil {
+		return nil, err
+	}
+
+	trb, err := store.GetTRBRequestByID(appcontext.ZLogger(ctx), feedback.TRBRequestID)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
From here to the end of this method, there are multiple DB updates - creating the feedback, updating the request, and possibly updating the request form (if edits are requested). I'm a little worried about an error partway through leaving the database in a partially-updated, inconsistent state; maybe we could create a utility method or two on `Store` that can wrap all the queries in a transaction?

Reply:
Working on it